### PR TITLE
feat: add minified `fuels` distribution

### LINKED ISSUE #1728: Add minified version in CDN
We should provide a minified version together with the normal one.

This is a follow-up issue on top of:
 - https://github.com/FuelLabs/fuels-ts/issues/1397

### PR DESCRIPTION
**Pull request by petertonysmith94**
- Relates to #1728

# Summary

- Added a minified version of the `fuels` distribution for use in CDNJS. I've used `uglifyjs` for minification based upon the following [benchmarks](https://github.com/privatenumber/minification-benchmarks/blob/master/README.md#-results). 

# Checklist

- [ ] I **_added_** — `tests` to prove my changes
- [ ] I **_updated_** — all the necessary `docs`
- [x] I **_reviewed_** — the entire PR myself, using the GitHub UI
- [ ] I **_described_** — all breaking changes and the Migration Guide

### CODE REVIEW

**Comment by petertonysmith94** [comment]
I will add relevant documentation once released to CDNJS.

File: .changeset/shiny-paws-chew.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"fuels": patch
+---
+
+feat: add minified `fuels` distribution
```
Comment (arboleya) [inline .changeset/shiny-paws-chew.md:5]: Is this still a WIP?

**Review by arboleya — DISMISSED** [review]

File: .changeset/shiny-paws-chew.md (hunk 1 above)
Comment (petertonysmith94) [reply .changeset/shiny-paws-chew.md:5]: Just wanted to test the minified distribution

**Review by danielbate — DISMISSED** [review]
LGTM. And apologies for ignorance with CDNJS, but is there any further integration needed there?

**Comment by petertonysmith94** [comment]
> LGTM. And apologies for ignorance with CDNJS, but is there any further integration needed there?

Yes, we will need to:
- Release new version with the minified bundle to NPM.
- Make to changes CDNJS configuration file ([fuels.json](https://github.com/cdnjs/packages/blob/master/packages/f/fuels.json)):
  ```json
  {
    ...,
    "autoupdate": {
      "source": "npm",
      "target": "fuels",
      "fileMap": [
        {
          "basePath": "dist",
          "files": ["browser.mjs", "browser.min.mjs"]
        }
      ],
      "ignoreVersions": [
        "0.0.0-*"
      ]
    },
    ...,
  }
  ```
  (Add minified bundle (`browser.min.mjs`) and `ignoreVersions`)

**Comment by danielbate** [comment]
> Yes, we will need to:
> 
> Release new version with the minified bundle to NPM.
> Make to changes CDNJS configuration file ([fuels.json](https://github.com/cdnjs/packages/blob/master/packages/f/fuels.json))

Should we not close #1728 till that is done?

**Review by arboleya — APPROVED** [review]

**Review by danielbate — APPROVED** [review]

**Review by danielbate — APPROVED** [review]

**Review by arboleya — APPROVED** [review]

**Review by maschad — APPROVED** [review]

**Pull request merged**